website/docs/docs/build/build-semantic-layer-intro.md

```diff
@@ -0,0 +1,56 @@
+---
+title: "Build your Semantic Layer"
+id: build-semantic-layer-intro
+description: "Learn about MetricFlow and build your Semantic Layer with Semantic models and Metrics"
+sidebar_label: Build your Semantic Layer
+tags: [Metrics, Semantic Layer]
+hide_table_of_contents: true
+---
+
+Seamlessly build your semantic layer using [semantic models](/docs/build/semantic-models) and [metrics](/docs/build/metrics-overview), avoid repetition, and  optimize your workflow with concepts you're already familiar with using dbt metrics, powered by **MetricFlow**. 
```

Comment:
This sentence reads a bit weird. If we're going to use a tricolon (1,2, & 3) can we hammer home the final point?

Reply:
gotcha, what about this suggest below?